src/content/pages/security/bug-bounty.md

```diff
@@ -0,0 +1,12 @@
+---
+header:
+  title: "Bug Bounty Program"
+  description:
+    [
+      "Filecoin Foundation is deeply committed to the integrity and security of the Filecoin network. With that in mind, we launched a bug bounty program for reported security vulnerabilities on the Filecoin protocol. Earn up to 150,000 USD, paid in USD/USDC, for reporting critical vulnerabilities",
+      "Since launching our bug bounty program, we have worked with over 100 researchers and paid out more than $400,000 in rewards –– recognizing external efforts in strengthening our network’s security.",
+    ]
+seo:
+  title: "Bug Bounty Program"
```

Comment:
Not sure where to check, is this what we're putting as a title? Coordinated Disclosure Policy has "Filecoin Bug Bounty Program - Disclosure Policy" - I assumed we should include "Filecoin Bug Bounty Program", but just a guess

Reply:
I opened a ticket for this - It's something to check with David + Jordan once the pages are up [>>](https://www.notion.so/filecoin/FF-SEO-Title-and-Descriptions-for-New-Security-pages-Digest-page-3829a37b83bd4d3baa9eb9d6f81e4021?pvs=4)